OpenRA.Mods.Common/Traits/Render/RenderDebugState.cs

```diff
@@ -65,7 +65,7 @@ void INotifyOwnerChanged.OnOwnerChanged(Actor self, Player oldOwner, Player newO
 
 		Color GetColor()
 		{
-			return self.EffectiveOwner != null ? self.EffectiveOwner.Owner.Color.RGB : self.Owner.Color.RGB;
+			return self.EffectiveOwner != null && self.EffectiveOwner.Owner != null ? self.EffectiveOwner.Owner.Color.RGB : self.Owner.Color.RGB;
```

Comment:
This should be looking at `EffectiveOwner.Disguised` for consistency with the other checks.